jre_emul/android/platform/libcore/harmony-tests/src/test/java/org/apache/harmony/tests/java/util/zip/InflaterInputStreamTest.java

```diff
@@ -18,19 +18,22 @@
 
 import java.io.ByteArrayInputStream;
 import java.io.ByteArrayOutputStream;
-import java.io.EOFException;
-import java.io.IOException;
-import java.io.InputStream;
 import java.io.File;
 import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
 import java.util.zip.DeflaterOutputStream;
 import java.util.zip.Inflater;
 import java.util.zip.InflaterInputStream;
-
-import junit.framework.TestCase;
+//import libcore.junit.junit3.TestCaseWithRules;
```

Comment:
Follow the convention commenting out lines. Same below for the rest of the file.

Reply:
added J2ObjC comments